scripts/upload_ceph_images_to_nexus.sh

```diff
@@ -330,9 +330,16 @@ function exit_maintenance_mode() {
   # shellcheck disable=SC2086
   if [[ "$(ceph orch host maintenance exit $node)" ]]; then
     # shellcheck disable=SC2086
+    counter=0
     until [[ "$(ceph orch host ls --host_pattern $node --format json-pretty|jq -r '.[].status')" != "maintenance" ]]; do
       echo "Waiting for node $node to exit maintenance mode."
       sleep 15
+      counter=$(( $counter + 1 ))
```

Comment:
i think the MIDDLE $counter works fine as just counter (shellcheck would normally report this as an issue). However, works fine as is.

Reply:
It seems to work both ways but I changed it to `$(( counter + 1 ))` since that is the better way.